tools/topology/topology2/include/components/gain.conf

```diff
@@ -154,4 +154,5 @@ Class.Widget."gain" {
 	curve_type		"linear"
 	curve_duration		10
 	init_value		0x7fffffff
+	num_audio_formats 1
```

Comment:
please check comments by @ranj063. It is not good to set it in gain.conf but should be in pipeline.conf

Reply:
> please check comments by @ranj063. It is not good to set it in gain.conf but should be in pipeline.conf

Please ignore the commit. It was pushed by mistake. :)